packages/nx/src/project-graph/affected/locators/workspace-projects.ts

```diff
@@ -31,6 +31,10 @@ export const getImplicitlyTouchedProjects: TouchedProjectLocator = (
   const globalFiles = [
     ...extractGlobalFilesFromInputs(nxJson, projectGraphNodes),
     'nx.json',
+    'package-lock.json',
```

Comment:
This is better than it not working... but also means that updating any dependency will cause every project to be affected...